packages/next/server/font-utils.ts

```diff
@@ -136,17 +163,16 @@ export function calculateSizeAdjustValues(fontMetrics: any) {
     ascent,
     descent,
     lineGap,
-    fallbackFont: fallbackFont.name,
+    fallbackFont: fallbackFontMetrics.familyName,
     sizeAdjust: formatOverrideValue(sizeAdjust),
   }
 }
 
 function calculateOverrideCSS(font: string, fontMetrics: any) {
   const fontName = font.trim()
 
-  const { ascent, descent, lineGap, fallbackFont } = calculateOverrideValues(
-    fontMetrics[fontName]
-  )
+  const { ascent, descent, lineGap, fallbackFont } =
+    calculateOverrideValues(fontName)
```

Comment:
@janicklas-ralph not sure if you were planning to, but you should be able to replace the majority of the code in this file with the `createFontStack` API coming to the core package:

```ts
import { createFontStack } from '@capsizecss/core';
```

Snapshot details: @capsizecss/core@0.0.0-create-font-stack-20221117062509
Documentation: [Link](https://github.com/seek-oss/capsize/blob/create-font-stack/README.md#createfontstack).


That would see the `getFontOverrideCss` function look something like this:
```ts
  try {
    const fontNames = parseGoogleFontName(css)

    const fontCss = fontNames.reduce((cssStr, fontName) => {
      const fontMetics = require(`@capsizecss/metrics/${formatName(fontName)}`);
      const fallbackMetrics = fontMetrics.category === 'serif'
        ? require(`@capsizecss/metrics/timesNewRoman`)
        : require(`@capsizecss/metrics/arial`);
      
      const { fontFaces, fontFamily } = createFontStack([ fontMetrics, fallbackMetrics ]);
      
      // Not sure where the computed `fontFamily` is being managed?
      
      cssStr += fontFaces
      return cssStr
    }, '')
    return fontCss
  } catch (e) {
    console.log('Error getting font override values - ', e)
    return ''
  }
```

Reply:
Given we are still experimenting with the different metric strategies, you will need to normalise the one you want into the `xWidthAvg` property before passing it into `createFontStack`:

For example:
```ts
const { fontFaces, fontFamily } = createFontStack([ 
  { ...fontMetrics, xWidthAvg: fontMetrics.xAvgLetterFrequency },
  { ...fallbackMetrics, xWidthAvg: fallbackMetrics.xAvgLetterFrequency }
]);
```

This complexity goes away once we settle on which strategy performs best.